dnsext-utils/DNS/TAP/Schema.hs

```diff
@@ -15,6 +15,7 @@ module DNS.TAP.Schema (
     composeMessage,
     SocketFamily (IPv4, IPv6),
     SocketProtocol (UDP, TCP, DOT, DOH, DNSCryptUDP, DNSCryptTCP, DOQ),
+    HttpProtocol (HTTP_NONE, HTTP1, HTTP2, HTTP3),
```

Comment:
You can write `HttpProtocol (..)` since this is not pattern synonyms.

Reply:
fixed, thanks.